### ISSUE TITLE
Enhancing Discover and sense of activity

### ISSUE THREAD

**Issue by vazquez** [mozilla/webmaker-core#640]
Elemental enhancements are those small changes made to a product that may change the way a user understands or uses the app, and thereby gives us information about that change.

These enhancements should be small design and development tasks. With analytics, we can test if these changes have been positive.

These are some ideas:
- Add a "New" label to projects that have been recently featured
- Add a counter to the user's profile that states how many projects they have published. "You've created 3 projects, so far. Keep it up!"
- Add an avatar besides the name of the project in Discovery view
- "100 people have viewed your project!" notification
- Add Project titles to header
- Double-tapping on an element immediately enters Edit mode
- Background coloured text
- Add a ✓ on Editing Options instead of using <- button as confirmation. The mental model is backwards.

**Comment by vazquez** [comment]
cc @xmatthewx

**Comment by xmatthewx** [comment]
Can we define the problem better? For example, there isn't a strong sense of activity ... what are elemental interventions that could change that?

I'm intrigued with notifications. A full-fledged system would require a big lift, but maybe there a small tests we can run in the make view, like view count like you mention. Another: If a project is featured in discover, can we mark it as such in the make view?

**Comment by ryanwarsaw** [comment]
Another idea I'd like to emphasis, is having time stamps. Time stamps for when I created the project, when it was featured, etc. You'd be surprised how helpful things like that can be, especially if you are trying to determine a tutorial/guides relevancy in comparison to the latest technology.

TLDR; Lets say I'm using Webmaker to learn how to use Blender (3D modelling tool), I find a guide on Webmaker to learn how to use this tool. The guide is from lets say 6 months ago, a new release came out 2 months ago. I can assume that some of that guide is out of date.

**Comment by ryanwarsaw** [comment]
I like the idea of counting views, not sure if I'm a big fan of alerting users for hitting milestones. I think it would definitely be cool if we alerted users when their project hit the discovery page, however. It would be neat if we modified the projects settings tab to include analytics for their project, total view count, some neat graphs showing the demographics, etc.

**Comment by ryanwarsaw** [comment]
As for displaying the project title in the header, I am working towards this, but its definitely a challenge and will require a bit of work. Currently, we are drawing and building the header in Android natively, so we'd have to add some new methods in the JS bridge for setting the header. Which comes with its own set of unique challenges as well.

**Comment by vazquez** [comment]
Other thoughts: Process of getting something featured is weird. As soon as you make something, it's considered published. Should we switch that mental model?

**Comment by vazquez** [comment]
#### Problem

There is no sense of activity in Webmaker app.

#### Why?

Users are not connected to each other. It seems like they are creating pages for themselves, when they really are creating content for a whole community.

#### What are the Webmaker social features already implemented?
- Remixing is a highly social feature, and one that Webmaker uses well. 
- Discover Gallery is another social feature, where users can see projects that have been curated and selected by Mozilla staff.
- Well, sharing! Using email, texting, or using Facebook to share a Webmaker project with someone.

#### Possible Elemental Enhancements

There are some enhancements we can do to:
1. Increase the sense of activity across Webmaker (ie. you're not alone in this app).
2. Make the act of remixing, discovery, and sharing more social.

##### Option A

We can propagate these elements on the user's Make view in the form of a badge/icon/something:
- Editor's Choice: When we feature a project on Discover Gallery.
- Fan favourite: When a lot of users have viewed the project.
- Remix'ed!: Someone has remixed their project, let's celebrate!

##### Option B

We can use the Profile section to display the most basic stats of the projects and how they have affected the Webmaker community.

This is the profile section we can improve to reflect more activity:

![screenshot 2015-09-09 12 44 23](https://cloud.githubusercontent.com/assets/1119821/9768265/85d2293c-56f0-11e5-86a3-90d4848759a1.png)

##### Option C

Taking on what Slack has integrated in their product, we can bring reactions to Webmaker! Other benchmarks such as Byte are doing this in a way. In our model, users on the app can leave a reaction to a person's project. Maybe this can be emoji? Who knows!

<img width="237" alt="screenshot 2015-09-09 13 03 32" src="https://cloud.githubusercontent.com/assets/1119821/9768752/b49ff35e-56f3-11e5-9f50-23b3c33e8d44.png">

#### Option D

Should we add likes? One of the universal features of social systems is the concept of the big ol' :heart:. What about displaying likes in Discover Gallery as a means of gauging project interest?

**Comment by xmatthewx** [comment]
I like the focus on a sense of activity and discover. 

Option A & B - These seem like good, low risk ideas. Should be designed so all users have some info even if they have low to no activity. First step would be to make sure we're counting views and remixes in the DB, a feature we need anyhow.

Option C - this would be valuable, but is a bigger lift, both in design and engineering. 

Option D - likes would be more useful if users could access a broader range of projects.

**Comment by xmatthewx** [comment]
A few more ideas:
- Discover show more - infinite scroll or a load more button at the bottom would increase access to projects
- Author view - link author names from discover to an author project list view. this would allow users wander and explore 
- Browser-based Discover view - could be built as a sidebar to project display. Or could be a standalone page. This won't have immediate impact on new users. Probably less valuable.
- Hashtags - a bit more of a lift, but would enable people to find things of interest across Webmaker

**Comment by lauradereynal** [comment]
This topic came back as a very important point with 2 users today. They phrased it asl
- How do I see who has viewed my project?
- How does this person knows that I have remixed their project ? How can they see my remix?
- How do I share my project ? Why is it not published yet ? How do you make it public ? (similar to another ticket I filed last week)
- How do I get my project in here?(discover gallery)

**Comment by lauradereynal** [comment]
- How do you communicate with people ?
- How do i get people to interact with me if they like my art ?
